Methods.md

```diff
@@ -1,39 +1,49 @@
 ## 1 Methods
 
 ### 1.0 Naming
+
 Method names should be a summary of what the method is doing, it needs to stay percise and short and representative of the operation with respect to synchrony.
 
 #### 1.0.0 Verbs
+
 Method names must contain verbs in them to represent the action it performs.
+
 ##### Do
+
 ```cs
 public List<Student> GetStudents()
 {
-	...
+    ...
```

Comment:
C# coding style by Microsoft prefers spaces rather than tabs. (If you use other style-guide that uses tabs, I'll revert this)

Reply:
Can you reference the style that prefers spaces over tabs?

Reply:
https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md

> We use four spaces of indentation (no tabs).

Reply:
They prefixed that with: "The general rule we follow is "use Visual Studio defaults"." which is a tab, which is what we will be following for this standard.

Reply:
Visual Studio defaults is spaces I think.
From the menu bar, **Tools** > **Options** > Expand **Text editor** > Expand **C#** > Open **Tabs** node.
You'll see that **Tab Size** is set to **4**, and **Insert spaces** is selected rather than **Keep tabs**.

But, anyways, if you'd like to keep tabs, I'll revert.

Reply:
Yeah, let's keep tabs, make sure you have no conflicts.